CHANGELOG.next.toml

```diff
@@ -11,6 +11,12 @@
 # meta = { "breaking" = false, "tada" = false, "bug" = false }
 # author = "rcoh"
 
+[[smithy-rs]]
+message = "Remove Unknown variant from enums in server"
+references = ["smithy-rs#1187"]
+meta = { "breaking" = true, "tada" = false, "bug" = false }
+author = "82marbag"
```

Comment:
We do not publish CHANGELOG entries for server only changes. If this PR does not change how the client SDK is generated, this should go away.